pytest_plugins/marker_deselection.py

```diff
@@ -47,7 +47,7 @@ def pytest_collection_modifyitems(items, config):
         # Include/Exclude tests those are not part of SatQE CI
         item_component = item.get_closest_marker('component')
         if item_component and (item_component.args[0] in non_satCI_components):
-            if item_component.args[0] in include_non_satci_tests:
+            if item_component.args[0] in include_non_satci_tests or item.nodeid.startswith('tests/upgrades/'):
```

Comment:
Does it really only select *Virtwho* upgrade tests? As opposed to all upgrade tests.

Reply:
yes, becasue non_satCI_components only has `VirtWho` tests in the list.